src/Robin/Ntlm/Crypt/Random/OpenSslRandomByteGenerator.php

```diff
@@ -15,7 +15,7 @@
  * "openssl" extension.
  *
  * @link http://php.net/openssl
- * @deprectated NOTE! This implementation is deprecated, as it's been found to
+ * @deprecated Insecure! This implementation is deprecated, as it's been found to
```

Comment:
Why the change to "insecure!"? The " NOTE!" is a common "task tag" (along with "TODO" and "FIXME") that gets automatically highlighted by many text editors. Changing it to " insecure!" removes that functionality.